docs/src/api/class-frame.md

```diff
@@ -97,6 +97,33 @@ with sync_playwright() as playwright:
     run(playwright)
 ```
 
+```csharp
+using Microsoft.Playwright;
+using System;
+using System.Threading.Tasks;
+
```

Comment:
Do you think we need to add the using in examples? just wondering.

Reply:
This applies to the rest of the open PRs

Reply:
I mimic the other languages. Some examples are "full", and it feels natural for us to do the same. *Ideally* down the road, we'll have a "Run this example" somewhere, but... until then... I'm up for consistency.